src/Arcus.Testing.Assert/AssertCsv.cs

```diff
@@ -74,6 +75,21 @@ public AssertCsvOptions IgnoreColumn(string headerName)
         /// </summary>
         internal IReadOnlyCollection<string> IgnoredColumns => _ignoredColumns;
 
+        /// <summary>
+        /// Adds a column via a zero-based index which will get ignored when comparing CSV tables.
+        /// </summary>
+        /// <param name="index">The zero-based index of the column that should be ignored.</param>
+        public AssertCsvOptions IgnoreColumnIndex(int index)
```

Comment:
We will also need to change the (preview) documentation in this PR, with that we now support column indexes. 
That would be this file: `/docs/preview/02-Features/02-assertion.md` Thx! 👍

Reply:
Docs updated in [9eeb4d9](https://github.com/arcus-azure/arcus.testing/pull/161/commits/9eeb4d9ec233d0d4939538257a869078f98b4078)